lib/formatFn.js

```diff
@@ -0,0 +1,49 @@
+'use strict'
+
+const countSpecifiers = require("./countSpecifiers")
+const specifiersRE = /(%[sdifjoOc%])/g
+
+function formatFn(message) {
+  const amountOfPlaceholders = countSpecifiers(message)
+
+  if (amountOfPlaceholders === 0) {
+    return () => message
```

Comment:
maybe just use string variable in `FastifyError`

Reply:
More likely to build FastifyError dynamically.

Reply:
I didn't get it

Reply:
nevermind. 

Anyhow. this is just a poc. If there is interest, I could improve it.